### ISSUE TITLE
Keybindings are off for non QWERTY keyboard layouts

### ISSUE THREAD

**Issue by otbe** [warpdotdev/Warp#264]
### Describe the bug

```shell
I don't know if this is related to the missing option of customizing shortcuts (which already exists as an issue) but on my QWERTZ (german) keyboard layout Im not able to increase the font size via shortcut. It says `0` for reset and `=` for increase but thats actually the same key. 😅
If I switch to standard qwerty it works as expected.
```

### To Reproduce

Steps to reproduce:
2. Click on '...'
3. Scroll down to '...'
4. See error '...'

### Expected behaviour

```shell
Would be nice to have the same way of font resizing like in any other app (cmd+'+' cmd+'-' and cmd+'0')
```

### Operating System

MacOS

### OS Version

11.6

### Additional context

```shell
Thanks for having me here :) love it so far!
```

**Comment by prog666** [comment]
I get the same issue when using dvorak-qwerty layout, it's supposed to have same keybindings as regular qwerty when used with `command` key, but keybindings are off, as if the actual letter is used, and not the key

**Comment by gershik** [comment]
I have the same issue with a Cyrillic keyboard layout. Keyboard shortcuts that require Latin characters don't work at all.

**Comment by lingdocs** [comment]
Yes, I am having the [same issue](https://github.com/warpdotdev/Warp/issues/264#issuecomment-1016420118) as @dderg on Mac OS 12.2.1 using the Dvorak - QWERTY ⌘ layout.

For other apps when I use `command ⌘` + a key it works as a QWERTY keyboard while holding down the `command ⌘` key.  But with warp unfortunately this doesn't work and I have to keep remembering to go back to full Dvorak mode in my head while holding the `command ⌘` mode. iTerm doesn't have this problem so I think I'll have to keep using that until this issue is resolved.

**Comment by elviskahoro** [comment]
> AI experimental text to command keymap, ctrl+`, isn't possible on mac's azerty (pc) keyboard mode.

**Comment by FlorianRaichle** [comment]
I have a similar issue, when trying to open the AI command search on a qwertz keyboard. The backtick is only accessible through pressing shift and so the command doesn't work.

**Comment by japsu** [comment]
I recently started experiencing a similar issue: previously history search would be triggered by Ctrl+R according to my current keyboard layout, now it is triggered only by Ctrl+R according to QWERTY.

**Comment by Nitive** [comment]
The problem seems to be fixed for me with latest release

**Comment by elviskahoro** [comment]
@japsu @FlorianRaichle is it working for yall as well
Please comment and I'll reopen!

**Comment by japsu** [comment]
In v0.2022.08.08.08.21.stable_00, I am still experiencing this issue.

> I recently started experiencing a similar issue: previously history search would be triggered by Ctrl+R according to my current keyboard layout, now it is triggered only by Ctrl+R according to QWERTY.

**Comment by elviskahoro** [comment]
Thanks @japsu
